x-pack/plugins/cases/server/common/models/case_with_comments.ts

```diff
@@ -331,6 +338,38 @@ export class CaseCommentModel {
     await this.params.services.alertsService.updateAlertsStatus(alertsToUpdate);
   }
 
+  private async updateAlertsSchemaWithCaseInfo(alertAttachments: CommentRequestAlertType[]) {
```

Comment:
What do you think about moving this function into the AlertsService? An added benefit I think is that we could initialize the AlertsService with the AlertClient in the client factory class and then we don't have to plumb it through to this class.